### PR TITLE
consultancycontroller and homepage created

### PR DESCRIPTION
**Pull request by eyobiel**
One page consultancy url (home page) created with consultancy controller, index.html.twig  and base.html.twig created

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+!/var/logs
+/var/logs/*
+!var/logs/.gitkeep
+!/var/sessions
+/var/sessions/*
+!var/sessions/.gitkeep
+!var/SymfonyRequirements.php
+/vendor/
+/web/bundles/
+/nbproject/private/
```
Comment (reyostallenberg) [inline .gitignore]: Should be added as global gitignore and not for this project, see https://help.github.com/articles/ignoring-files/ 
Not everybody uses netbeans, and we don't want every environment setup representated in the projects .gitignore

File: nbproject/project.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
```
Comment (reyostallenberg) [inline nbproject/project.properties]: File should not be in this commit

File: nbproject/project.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (reyostallenberg) [inline nbproject/project.xml]: File should not be in this commit

File: src/AppBundle/Controller/ConsultancyController.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
```
Comment (reyostallenberg) [inline src/AppBundle/Controller/ConsultancyController.php]: Correct doc blocks

File: src/AppBundle/Controller/ConsultancyController.php [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+/**
+ * Description of ConsultancyController
+ *
+ * @author gebruiker
+ */
+class ConsultancyController extends Controller {
+    //put your code here
```
Comment (reyostallenberg) [inline src/AppBundle/Controller/ConsultancyController.php]: Remove this

File: .gitignore (hunk 1 above)
Comment (eyobiel) [inline .gitignore]: Hello Reyo, thanks for all the reviews. I just read the /help.github you sent me. So now if i configure the git in my bash through "git config --global core.excludesfile ~/.gitignore_global". Does it mean when i add the new changed or added files to my branch, the .gitignore file will be global?

File: nbproject/project.properties (hunk 2 above)
Comment (eyobiel) [inline nbproject/project.properties]: Whenever i add or change the files in my project, the two files (nbproject/project.properties and nbproject/project.xml) show up when i use git status. so does it mean i have to ignore them or not add them.

Comment (reyostallenberg) [inline nbproject/project.properties]: you should execute `git remove nbproject`

File: .gitignore (hunk 1 above)
Comment (reyostallenberg) [inline .gitignore]: you should only add `/nbproject/` to `~/.gitignore_global` file and remove it in this projects `.gitignore`

Comment (reyostallenberg) [inline .gitignore]: It means that in no repository you will see any tracking on /nbproject/

File: nbproject/project.properties (hunk 2 above)
Comment (reyostallenberg) [inline nbproject/project.properties]: if you add/correct the global gitignore you will not see these files on git status anymore

Comment (eyobiel) [inline nbproject/project.properties]: thanks i edited some of the files, when i was adding them i didn't see the nbprojects on my status

File: .gitignore [hunk 6]
```diff
@@ -11,7 +11,3 @@
 !/var/sessions
 /var/sessions/*
 !var/sessions/.gitkeep
-!var/SymfonyRequirements.php
-/vendor/
```
Comment (reyostallenberg) [inline .gitignore]: This should not be removed

File: .gitignore [hunk 7]
```diff
@@ -11,7 +11,3 @@
 !/var/sessions
 /var/sessions/*
 !var/sessions/.gitkeep
-!var/SymfonyRequirements.php
-/vendor/
-/web/bundles/
```
Comment (reyostallenberg) [inline .gitignore]: This should not be removed

File: .gitignore [hunk 8]
```diff
@@ -11,7 +11,3 @@
 !/var/sessions
 /var/sessions/*
 !var/sessions/.gitkeep
-!var/SymfonyRequirements.php
```
Comment (reyostallenberg) [inline .gitignore]: I think this line should not be removed

File: app/Resources/views/base.html.twig [hunk 9]
```diff
@@ -10,52 +10,57 @@
         {% endblock %}
         <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
     </head>
-    <body>
```
Comment (reyostallenberg) [inline app/Resources/views/base.html.twig]: Add indentation, to <body> and all nodes in <body>

File: app/Resources/views/base.html.twig [hunk 10]
```diff
@@ -10,52 +10,57 @@
...
         {% block body %}{% endblock %}
-       
+
         {% block javascripts %}
-             <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
-            <script src="{{ asset('js/main.js') }}"></script>
         {% endblock %}
     </body>
     <hr>
     <footer class="companyfooter">
```
Comment (reyostallenberg) [inline app/Resources/views/base.html.twig]: Everything visible (so also <footer>) should be inside <body>

File: app/Resources/views/base.html.twig [hunk 11]
```diff
@@ -10,33 +10,30 @@
...
-        {% block javascripts %}
-        {% endblock %}
+    <body>
+        <div class="container">
+            <header class="header">
+                <img class="logo-icon" src="{{ asset('images/image1.jpg') }}">
+                <h3 class=" companylogo logo"><small class= text-muted> Consultancy Agency</small></h3>
+            </header>
+        </div>
+            <nav class="navbar navbar-light bg-success" role ="navigation">
```
Comment (reyostallenberg) [inline app/Resources/views/base.html.twig]: All indentation should be correct, please fix it all

Comment (eyobiel) [inline app/Resources/views/base.html.twig]: alright will fix them

File: .gitignore [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
-!/var/logs
-/var/logs/*
-!var/logs/.gitkeep
-!/var/sessions
-/var/sessions/*
-!var/sessions/.gitkeep
-!var/SymfonyRequirements.php
-/vendor/
-/web/bundles/
+/app/config/parameters.yml
```
Comment (reyostallenberg) [inline .gitignore]: The `^M` should not be here... please read this http://stackoverflow.com/questions/1889559/git-diff-to-ignore-m/1889699#1889699 and correct it

File: src/AppBundle/Controller/ConsultancyController.php [hunk 13]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AppBundle\Controller;
+/**
```
Comment (reyostallenberg) [inline src/AppBundle/Controller/ConsultancyController.php]: This block should be under the `use` statements, also don't add more than 1 empty line under the `use` statements

File: web/stylebox/bootstrap/css/bootstrap-theme.css [hunk 14]
```diff
@@ -0,0 +1,587 @@
+/*!
```
Comment (reyostallenberg) [inline web/stylebox/bootstrap/css/bootstrap-theme.css]: Is it needed to have the bootstrap and fontawesome sources in this repository or can you make it installable? Normally we only add dists (minified) and even that is doubtful.

File: app/Resources/views/base.html.twig [hunk 15]
```diff
@@ -1,13 +1,65 @@
...
+                <li><a href="agribusiness">Agribusiness Management</a></li>        
+                <li><a href="contactform">Contact</a></li>        
+            </ul>
+        </div>
+    </nav>
         {% block body %}{% endblock %}
-        {% block javascripts %}{% endblock %}
-    </body>
-</html>
+</body>
```
Comment (reyostallenberg) [inline app/Resources/views/base.html.twig]: Ending body here is still incorrect, footer tag should be inside body tag

**Pull request merged**